src/Socket/messages-recv.ts

```diff
@@ -42,20 +42,24 @@ export const makeMessagesRecvSocket = (config: SocketConfig) => {
 
 	const appStateSyncTimeout = debouncedTimeout(
 		6_000,
-		async() => {
-			logger.info(
-				{ recvChats: Object.keys(recvChats).length },
-				'doing initial app state sync'
-			)
-			if(ws.readyState === ws.OPEN) {
-				await resyncMainAppState(recvChats)
-			}
-
-			historyCache.clear()
-			recvChats = { }
+		() => {
+			timeoutAppState().catch(err => onUnexpectedError(err, 'timing out app state'))
 		}
 	)
 
+	async function timeoutAppState() {
```

Comment:
```suggestion
	async function resyncAppState() {
```

since the action is resyncing (?) rather than timing out